mesh/tags/models.py

```diff
@@ -1,9 +1,14 @@
 from django.db import models
+from mesh.accounts.models import Account
 
 # Create your models here.
 class Tag(models.Model):
-    #TODO
-    pass
+    tagID = models.AutoField(primary_key=True)
+    tagName = models.CharField(max_length=255)
+    isDefault = models.BooleanField()
+
 class TagBridge(models.Model):
-    #TODO
-    pass
\ No newline at end of file
+    class Meta:
```

Comment:
I'm a little confused about the inclusion about the Meta class? Where is it being used? Do you really need it?

Reply:
@GuyWhoCode From what I can tell, [the Meta class](https://docs.djangoproject.com/en/4.2/ref/models/options/) just represents metadata for models, so while it doesn't seem explicitly necessary, it does seem that it can be useful purely for information. as something like unique_together is a metadata field provided by the Meta class. Although it might look a little more clear in code if the class Meta on line 11-12 was placed after the tagID and accountID definitions.